packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -1771,6 +2023,7 @@ export interface NormalizedVirtualOptions {
  * Return type is mostly useful for setups that rely on types like GraphQL.
  *
  * @example <caption>Checking password length before hashing it</caption>
```

Comment:
![image](https://user-images.githubusercontent.com/13023439/212564372-d22e1f02-a54d-4a96-acc4-f968e6933e8d.png)
This is still a bit broken